runtime/queries/metricsview.go

```diff
@@ -25,6 +25,8 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+type forExportKey struct{}
+
```

Comment:
Can we pass this as an explicit parameter or property instead? Context values are similar to reflection, best only to use them if truly necessary